cli/config/config.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"go.dedis.ch/kyber/v3/group/edwards25519"
+	"go.dedis.ch/kyber/v3/util/key"
+)
+
+func main() {
+	pair := key.NewKeyPair(&edwards25519.SuiteEd25519{})
+
+	fmt.Printf("PUBLIC_KEY=%s\nPRIVATE_KEY=%s\n", pair.Public, pair.Private)
```

Comment:
Minor readability pet peeve: I'd consider doing two `Println` 

But yes, this looks about right 👍

Reply:
I'll close it anyway and move it to the webapp itself.